pkgs/games/sopwith/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, glib
+, SDL2
+, libGL
+, pkg-config
+, autoconf
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sopwith";
+  version = "2.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/fragglet/sdl-sopwith/archive/refs/tags/sdl-sopwith-${version}.tar.gz";
+    hash = "sha256-3jIlB9AyWbevtYGEAIlNH6GIUKutEMRtRzay39ByMdg=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoconf
+    automake
+  ];
+
+  buildInputs = [
+    glib
+    SDL2
+    libGL
+  ];
+
+  preConfigure = "NOCONFIGURE=1 ./autogen.sh";
+
+  meta = with lib; {
+    homepage = "https://github.com/fragglet/sdl-sopwith";
+    description = "Classic biplane shoot ‘em-up game.";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Plus;
```

https://github.com/fragglet/sdl-sopwith/blob/trunk/COPYING.md and the header in various source files states gpl2 or later.